mj.config.cjs

```diff
@@ -201,10 +201,12 @@ const mjServerConfig = {
     autoCreateNewUsers: true,
     newUserLimitedToAuthorizedDomains: false,
     newUserAuthorizedDomains: [],
-    newUserRoles: ['UI'],
+    newUserRoles: ['UI', 'Developer'],
```

Comment:
@JS-BC  i still don't like auto-adding people to the Developer role. It makes me nervous for this to be the default. Remind me, why are we doing that? Maybe just make `UI` slightly more capable instead?

Reply:
If we modify the UI role to have more capabilities, then I think it should work like how the Basic role is defined in this issue: https://github.com/MemberJunction/MJ/issues/671